src/module-utils.js

```diff
@@ -0,0 +1,21 @@
+// a require function with both ES5 and ES6 default export support
+function requireX(path) {
+  const modul = require(path);
+  if (modul === null || modul === undefined) {
+    // if null do not bother
+    return modul;
+  } else {
+    if (
+      modul.__esModule === true &&
+      (modul.default !== undefined || modul.default !== null)
```

Comment:
This will always be true since `modul.default` can not be `undefined` and `null`.

Also this check should not be necessary. if a module wants to export `null` or `undefined` than I would rather the return be `undefined` or `null` than `{default: undefined}` or `{default: null}`.

If a module exports `__esModule = true` but doesn't provide a `default` export than this should return `undefined`.

Reply:
This check is necessary if the user has named their exports. 
```
export activate function {}
export deactivate function {}
```

We only want to use `default` if it is something other than undefined/null.

Reply:
true for packages but not in general. Maybe we do need two separate functions for our different use cases.

That line is still always `true`.

```suggestion
      (modul.default !== undefined && modul.default !== null)
```

Reply:
The code now works in those cases too. Unless literally, someone wants to export null! 😄